Packs/DemistoRESTAPI/ReleaseNotes/1_3_41.md

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
@ShirleyDenkberg, please review this file, and if anything needs to be modified, I'll update the others

Reply:
I reviewed that file.  See my comments.